### ISSUE TITLE
option to disable bandwidth test?

### ISSUE THREAD

**Issue by tobi-nmx** [akpw/mktxp#28]
It looks like mktxp measures internet bandwidth every 15sec from the Prometheus/exporter/Grafana machine (not from the Mikrotik router). Is there a way to disable the bandwidth check?

**Comment by akpw** [comment]
yes this was covered in [one of the prev threads](https://github.com/akpw/mktxp/issues/2)
